CogniteSdk/src/Resources/Units.cs

```diff
@@ -54,7 +54,7 @@ public async Task<IItemsWithoutCursor<UnitItem>> GetUnitAsync(string unitExterna
         /// <param name="ignoreUnknownIds">Ignore IDs and external IDs that are not found. Default: false</param>
         /// <param name="token">Optional cancellation token.</param>
         /// <returns>A sequence of the requested units.</returns>
-        public async Task<IEnumerable<UnitItem>> RetrieveUnitsAsync(IEnumerable<string> externalIds, bool? ignoreUnknownIds = null, CancellationToken token = default)
+        public async Task<ItemsWithIgnoreUnknownIds<UnitItem>> RetrieveUnitsAsync(IEnumerable<string> externalIds, bool? ignoreUnknownIds = null, CancellationToken token = default)
```

Comment:
this is a return type, I believe this was correct

Reply:
I got some LSP type mismatch errors with the previous return type, and that's why I changed it. I believe it was previously correct too, as we were returning `ret.Items` which is a `seq` in F# and a `IEnumerable` in C#. This change removes the auto expansion of `.Items` and returns  `ItemsWithIgnoreUnknownIds` instead.

I did a quick look over the other methods in other namespaces and found no consistency across: sometimes we reuturn an object with `.Items` and sometimes we return a list (where `.Items` have been expanded already). 

Since we were bumping a major version, I thought of refactoring the SdK to make the behaviour consistent, but I gave up after figuring out it would take too much effort to do so 😅. This small tweak on units was already commited by then, but I could revert it as well.

Reply:
I see the might be a bug with the previous implementation, as it sets `ItemsWithIgnoreUnknownIds` as a return type here `|> postV10<_, ItemsWithIgnoreUnknownIds<_>> request url`

However `ItemsWithIgnoreUnknownIds` should never be used as a return type (only as a query). 
If you get this `LSP type mismatch errors with the previous return type` I could have a look maybe?

Reply:
My bad, `ItemsWithIgnoreUnknownIds` should not be on the response indeed. Let me fix this to use `ItemsWithoutCursor`.